docs/src/componentDocs/DrawerFooter/playground/DrawerFooterTypes.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentType } from '../../../__types__';
+import * as Colors from '@brightlayer-ui/colors';
+
+export const drawerTypes: ComponentType = {
```

Comment:
I'm still a bit confused by all of these "Types". Did we decide to rename these to "Config" instead? e.g. drawerTypes => drawerConfig

Same goes for the file name.

If this still needs addressed across all components we can address it in a separate story.

Reply:
> Types

We didn't but I can make changes in next PR.